app/code/Bulbulatory/Recomendations/etc/webapi.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
```

Comment:
Ten plik pozwala na odpytywanie o różne obiekty za pomocą REST albo SOAP API z poziomu sklepu. Dodatkowo, ustawione jest aby dostępne było to dla każdego (`<resource ref="anonymous"/>`), co raczej nie jest pożądane - te dane powinny być widoczne tylko w panelu. W związku z tym można ten plik usunąć, aby nie powodował potencjalnych problemów z wyciekami wrażliwych danych itp.